OlympusDAO-Agent-Suit/Bond-OnlyPolicyMethods/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "bond-policy-agent",
+  "version": "0.0.1",
+  "description": "Agent checks policy function call on bond contracts.",
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest"
+  },
+  "dependencies": {
+    "ethers": "^5.5.2",
```

Comment:
Is this dependency needed? I think `ethers` is installed automatically when you initialize the agent